osbs/constants.py

```diff
@@ -112,3 +112,9 @@
 
 # optional key path for filtering existing build config results
 FILTER_KEY = 'spec.source.git.uri'
+
+# number of retries for git clone operations
+GIT_MAX_RETRIES = 3
+
+# backoff factor for git clone operations - exponential backoff in seconds
+GIT_BACKOFF_FACTOR = 5
```

Comment:
Wonder if 60s is a better value here. Each clone operation can take from several seconds up to a couple of minutes. The likely reason for failure is high load on the git server, so even the largest backoff of 40s seems pretty fast. Changing the backoff factor to 60 means we'll wait 15min in total which seems reasonable.